src/main/java/com/iexec/worker/tee/post/compute/App.java

```diff
@@ -16,7 +16,10 @@
 @Slf4j
 public class App {
 
-    public static void main(String[] args) {
+    private App() {
+    }
+
+    public static void main() {
```

Comment:
Why change java legal `main` method signature ?

Reply:
Auto-correction, fixed, thanks for heads-up!